src/test/locale/ru.js

```diff
@@ -353,3 +353,35 @@ test('weeks year starting sunday formatted', function (assert) {
     assert.equal(moment([2012,  0,  9]).format('w ww wo'), '3 03 3-я', 'Jan  9 2012 should be week 3');
 });
 
+test('parsing RFC 2822', function (assert) {
```

Comment:
It's confusing for this to be in the ru test file. You can test it where the rest of the RFC2822 tests are and just try/finally around `moment.locale('ru')` to show that it ignores the locale.